packages/manager/src/features/CloudPulse/shared/CloudPulseRegionSelect.tsx

```diff
@@ -56,7 +88,7 @@ export const CloudPulseRegionSelect = React.memo(
         loading={isLoading}
         noMarginTop
         placeholder={placeholder ?? 'Select a Region'}
-        regions={regions ? regions : []}
+        regions={supportedRegions ? supportedRegions : []}
```

Comment:
```suggestion
        regions={supportedRegions ?? []}
```
Nit: this could be simplified with the nullish coalescing operator